mpas_analysis/ocean/sst_timeseries.py

```diff
@@ -1,6 +1,7 @@
 import xarray as xr
 import pandas as pd
 import datetime
+from netCDF4 import Dataset as netcdf_dataset
```

Comment:
@pwolfram, this is not the level of review that I need of this code right now.  But I agree with the suggestion.  I, too, prefer camel-case but I don't now that we have established this as a style requirement of the repo, and if so it needs to be applied across the board.